Sources/PackageDescription/Version.swift

```diff
@@ -59,7 +92,39 @@ public func ==(v1: Version, v2: Version) -> Bool {
 extension Version: Comparable {}
 
 public func <(lhs: Version, rhs: Version) -> Bool {
-    return [lhs.major, lhs.minor, lhs.patch].lexicographicalCompare([rhs.major, rhs.minor, rhs.patch])
+    let lhsComparators = [lhs.major, lhs.minor, lhs.patch]
+    let rhsComparators = [rhs.major, rhs.minor, rhs.patch]
+    
+    if lhsComparators != rhsComparators {
+        return lhsComparators.lexicographicalCompare(rhsComparators)
+    }
+    
+    guard let lhsPrereleaseIdentifiers = lhs.prereleaseIdentifiers else {
+        return false // Non-prerelease lhs >= potentially prerelease rhs
+    }
+    
+    guard let rhsPrereleaseIdentifiers = rhs.prereleaseIdentifiers else {
+        return true // Prerelease lhs < non-prerelease rhs 
+    }
+    
+    for (lhsPrereleaseIdentifier, rhsPrereleaseIdentifier) in zip(lhsPrereleaseIdentifiers, rhsPrereleaseIdentifiers) {
+        if lhsPrereleaseIdentifier == rhsPrereleaseIdentifier {
+            continue
+        }
+        
+        let castedLhsIdentifier: Any = Int(lhsPrereleaseIdentifier) ?? lhsPrereleaseIdentifier
```

Comment:
Simple past tense is "cast". So `castLhsIdentifier` would be correct in a grammatical sense. (In case anyone actually cares )
Then again, I'm not a native speaker... ;)